hu/hu_usb.cpp

```diff
@@ -429,7 +429,7 @@ int HUTransportStreamUSB::Start(byte ep_in_addr, byte ep_out_addr) {
         {
             logw("Wating for the device to reconnect");
             //Give it some time to reconnect
-            ms_sleep(1000);
+            ms_sleep(2000);
```

Comment:
I have a code of one of my branches that basically just waits for a while in a loop enumerating USB devices until OAP device shows up. That made startup significantly faster, let me see if I'll manage to port it over this week.